src/main/java/com/alkemy/ong/service/UserService.java

```diff
@@ -32,12 +34,25 @@ public UserDTO registerUser(User user) {
         UserModel userModel = UserMapper.mapDomainToModel(user);
         userModel.setPassword(encryptPassword(user));
         UserModel save = userRepository.save(userModel);
-        return UserMapper.mapModelToDto(save);
+        User userDomain = UserMapper.mapModelToDomain(save);
+        return UserMapper.mapDomainToDTO(userDomain);
     }
 
     private String encryptPassword(User user){
         String password = user.getPassword();
         String encryptedPassword = passwordEncoder.encode(password);
         return encryptedPassword;
     }
+
+    @Transactional(readOnly = true)
+    public List<UserDTO> getAll(){
+        List<UserModel> userModelList = userRepository.findAll();
+        List<UserDTO> userDTOList = new ArrayList<>();
+        userModelList.forEach(userModel -> {
+            User userDomain = UserMapper.mapModelToDomain(userModel);
+            UserDTO userDTO = UserMapper.mapDomainToDTO(userDomain);
+            userDTOList.add(userDTO);
+        });
+        return userDTOList;
```

Comment:
Esto lo podes reemplazar para utilizar streams y map, y te queda asi (por si querés chusmear más):
```suggestion
        List<UserModel> userModelList = userRepository.findAll();
        return userModelList.stream().map(UserMapper::mapModelToDomain)
                .map(UserMapper::mapDomainToDTO)
                .collect(Collectors.toList());
```